README.md

```diff
@@ -2,14 +2,26 @@ _NOTE: The interfaces of flux-accounting are being actively developed and are no
 
 ## flux-accounting
 
-Development for a bank/accounting interface for the Flux resource manager. Writes and saves job step history, user account information, and priority calculation to persistent storage using Python's SQLite3 API. 
+Development for a bank/accounting interface for the Flux resource manager. Writes and saves user account information to persistent storage using Python's SQLite3 API.
 
-##### job step history
+##### Install instructions
 
-Running as a cron job, `write_jobs.py` uses a flux-core RPC to retrieve job data that have recently transitioned to an INACTIVE state in the past _x_ minutes. It parses the job record data and writes it to a SQLite database, which can be queried at a later time. 
+This repository uses `pandas` for data display information. To install this dependency, along with any other packages specified in **requirements.txt**, and also run unit tests, you can run `make check`:
 
-##### user account information
+```
+$ make check
+pip3 install -r requirements.txt
```

Comment:
When I used flux-sched docker, pip3 wasn't there so I had to manually install it. So you may as well mention pip3 as a dependency.